site/docs/providers/hyperbolic.md

```
@@ -16,35 +16,87 @@
 export HYPERBOLIC_API_KEY=your_api_key_here
 ```
 
-## Provider Format
+## Provider Formats
 
-The provider format is:
+### Text Generation (LLM)
 
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)